test/client/getOrderbook.ts

```diff
@@ -66,19 +67,9 @@ function normalRippledResponse(request: BookOffersRequest): object {
   ) {
     return rippled.book_offers.fabric.requestBookOffersAsksResponse(request)
   }
-  throw new Error('unexpected end')
+  throw new XrplError('unexpected end')
 }
 
-// function xrpRippledResponse(request: BookOffersRequest): object {
```

Comment:
This shouldn't be removed - it'll likely be useful when the `getOrderbook` tests are rewritten